examples/vault/demo.vault-setup.manual.sh

```diff
@@ -0,0 +1,133 @@
+
```

Comment:
shebang has to be on the first line i think? not sure what is the behavior otherwise